04-spatial-operations.Rmd

```diff
@@ -2,15 +2,16 @@
 
 ## Prerequisites {-}
 
-- This chapter requires the packages **tidyverse**, **sf** **raster**, and **spData**:
+- This chapter requires the packages **tidyverse**, **sf** and **raster**. 
+To avoid confusing function names (e.g., `dplyr::select` and `raster::select`) we will wait with attaching the **raster** package until we reach section \@ref{spatial-operations-on-raster-data):
 
 ```{r, message=FALSE}
 library(sf)
-library(raster)
 library(tidyverse)
+# library(raster)
 ```
 
-- Please attach also the **spData** package which automatically attaches the `world` dataset:
+- Please attach also the **spData** package which in turn automatically attaches the `world` dataset:
```

Comment:
I don't think the word 'please' is appropriate here, overpolite! (May have been my English politeness that introduced it but can't remember.)

Reply:
changed that